### PR TITLE
Add missing Create/Update/Delete API calls for Proca entities 

### PR DESCRIPTION
**Pull request by marcinkoziej**
Table of crud operations on our objects.

```
|-------------+-----+--------+-----------+--------+-----|
|             | Add | Update | Upsert    | Delete | Get |
| Campaign    | 1   | X      | X         | 2      | 4   |
| Action Page | X   | X      | X (via C) | 3?     |     |
| User        | I/A | X      |           | ?      |     |
| Staffer     | X   | X      |           | X      | X   |
| Group       |     |        |           |        |     |
| Org         | X   | X      |           | X      |     |
| Service     |     |        | X         |        |     |
| Key         | XG  | A      |           |        |     |
```
Legend:
- Key - can be added and (G)enerated, can be (A)ctivated
- 

Todo:

- [x] addCampaign (1) 
- [x] deleteCampaign (2)
- [x] deleteActionPage - removes AP if there is no live action data on it
- [x] normalize campaign getting / selecting

@tttp something missing?
@hiemanshu

### CODE REVIEW

File: proca/utils/create-admin-account.exs [hunk 1]
```diff
@@ -30,7 +30,7 @@ defmodule Proca.CreateAdminAccount do
     case user do
       nil -> IO.puts("I couldn't create or find a user with email \"#{email}\". Is it really an email?")
       %Proca.Users.User{} -> 
-        Proca.Users.User.update(user, [:admin])
+        Proca.Repo.update! Proca.Users.User.make_admin_changeset(user)
```
Comment (hiemanshu) [inline proca/utils/create-admin-account.exs:33]: Always use parentheses for args?

**Review by hiemanshu — COMMENTED** [review]
LGTM. Small style change questions

File: proca/lib/proca/confirm.ex [hunk 2]
```diff
@@ -113,45 +113,51 @@ defmodule Proca.Confirm do
...
     from(c in Confirm, where: c.code == ^code and c.email == ^email, limit: 1)
-    |> one()
+    |> Repo.one()
   end
 
   def reject(confirm = %Confirm{}, auth \\ nil) do
     confirm 
     |> change(charges: 0) 
-    |> update!
+    |> Repo.update!
```
[the remark this answers is not in the document]
Comment (marcinkoziej) [reply proca/lib/proca/confirm.ex:160]: Agree, let's use () even when empty.
Sorry there might be some older code that needs cleanup. 
Pity there is still no elixir prettyfier

File: proca/utils/create-admin-account.exs (hunk 1 above)
Comment (marcinkoziej) [reply proca/utils/create-admin-account.exs:33]: Yes

File: proca/lib/proca/action_page.ex [hunk 3]
```diff
@@ -59,6 +63,7 @@ defmodule Proca.ActionPage do
       :locale,
       ~r/^[a-z]{2}(_[A-Z]{2})?$/
     )
+    |> change(assocs)
```
[the remark this answers is not in the document]
Comment (marcinkoziej) [reply proca/lib/proca/action_page.ex:66]: The most idiomatic way to put assoc only when it is given

[the remark this answers is not in the document]
Comment (marcinkoziej) [reply proca/lib/proca/action_page.ex:67]: with put assoc, you need to call it conditionally depending on if association was given

**Pull request merged**